### PR TITLE
Added requests and lxml to the dependencies

### PR DESCRIPTION
**Pull request by 01110011011101010110010001101111**
Addressing the review issues mentioned in #7 by adding requests and lxml to the requirements.txt.

### CODE REVIEW

**Comment by cjquines** [comment]
will check 3.6 compat in a bit

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 beautifulsoup4==4.11.1
+lxml==4.9.1
+requests==2.28.1
 selenium==4.2.0
```
Comment (cjquines) [inline requirements.txt]: now that we've gotten rid of old scripts, i dont think we need selenium anymore

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
 beautifulsoup4==4.11.1
+lxml==4.9.1
+requests==2.28.1
```
Comment (cjquines) [inline requirements.txt]:
```suggestion
requests==2.27.1
```
2.28.1 doesnt support 3.6

File: requirements.txt (hunk 1 above)
Comment (01110011011101010110010001101111) [reply requirements.txt]: Cool, tested it without selenium and update.py works fine. Updated in [1e819dd](https://github.com/cjquines/firehose/pull/9/commits/1e819ddace36f4f5aaa043e52944fbec1d3d55a2).

File: requirements.txt (hunk 2 above)
Comment (01110011011101010110010001101111) [reply requirements.txt]: Updated in [1e819dd](https://github.com/cjquines/firehose/pull/9/commits/1e819ddace36f4f5aaa043e52944fbec1d3d55a2)

**Pull request merged**